JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R

```diff
@@ -1374,6 +1374,8 @@
 		nullInterval <- c(-Inf, 0)
 		stretch <- 1.32
 	}
+    
+    BF <- as.numeric(BF)
```

Comment:
Can you add a comment explaining this? We will need to provide a proper fix in the long term.